code/game/turfs/open/_open.dm

```diff
@@ -235,7 +235,6 @@
 	planetary_atmos = TRUE
 	initial_gas_mix = LAVALAND_DEFAULT_ATMOS
 	baseturfs = /turf/open/indestructible/hierophant
-	smoothing_flags = SMOOTH_CORNERS
 	smoothing_flags = SMOOTH_BITMASK
```

Comment:
why wasn't this picked up by linters wtf

Reply:
spaceman doesn't care cause it would kill downstreams